moto_sync_from_s3/Tiltfile

```diff
@@ -0,0 +1,107 @@
+load("ext://uibutton", "cmd_button", "location")
+
+EXTENSION_ROOT = os.getcwd()
+PROJECT_ROOT = config.main_dir
+
+REQUIRED_SECRETS = [
+    "AWS_ACCOUNT_ID",
+    "AWS_S3_ROLE_NAME",
+    "AWS_SSO_WEB_START_URL",
+]
+
+
+def moto_sync_from_s3(
+    bucket_name,
+    object_prefix = "/",
+    secrets_path = ".env",
+    button_location = location.NAV,
+):
+    """Sync s3 moto bucket from real s3 bucket.
+
+    Secrets should be prefixed with `S3SYNC_<source_bucket>__`, i.e.:
+      - S3SYNC_MYBUCKETNAME__AWS_CLIENT_SESSION_NAME
```

Comment:
Is there a particular reason why the env prefix requires the bucket name? This will end up in duplication for several variables since they are the same value regardless of bucket.  I think this should just work off one set of env variables instead of a set for each bucket that a project works with. 

For example in one project we work with 2 buckets, so id have to define 16 env variables when i could only do 8.